api/api.proto

```diff
@@ -4450,3 +4456,57 @@ enum EnforcementType {
   // This specifies that the rule is preferred and can be best effort
   preferred = 1;
 }
+
+enum RestoreParamBoolType {
+	PARAM_BKUPSRC = 0; // Default: whateever was cloudbakup's option for the parameter
+	PARAM_FALSE = 1;
+	PARAM_TRUE = 2;
+}
+
+// RestoreSpec allows some of the restore volume properties of to be modified
+// while restoring cloud baackup.
+message RestoreVolumeSpec {
+  // HaLevel specifies the number of copies of data.
+  int64 ha_level = 1;
+  // Cos specifies the relative class of service.
+  CosType cos = 2;
```

Comment:
Needs a value similar to io_profile to pick up from backup source?

Reply:
this field has None value that it can be set to if user wishes to fallback whats in backup's spec.